pkgs/applications/graphics/odafileconverter/default.nix

```diff
@@ -14,8 +14,8 @@ in mkDerivation {
 
   src = fetchurl {
     # NB: this URL is not stable (i.e. the underlying file and the corresponding version will change over time)
-    url = "https://download.opendesign.com/guestfiles/ODAFileConverter/ODAFileConverter_QT5_lnxX64_7.2dll.deb";
-    sha256 = "0sa21nnwzqb6g7gl0z43smqgcd9h3xipj3cq2cl7ybfh3cvcxfi9";
+    url = "https://download.opendesign.com/guestfiles/Demo/ODAFileConverter_QT5_lnxX64_7.2dll_21.9.deb";
```

Comment:
404 file not found.

Reply:
Seems that these URL's break very quickly; old version is deleted when a new version is released. What should we do? Have a mirror somehow?

Reply:
Some packages like this are mirrored on web.archive.org.

Reply:
Great idea. I did exactly that.